Library/Homebrew/requirements.rb

```diff
@@ -125,3 +125,9 @@ class GitRequirement < Requirement
   default_formula "git"
   satisfy { Utils.git_available? }
 end
+
+class NodeRequirement < Requirement
+  fatal true
+  default_formula "node"
+  satisfy { which("node") }
+end
```

Comment:
Please add this to a separate `node_requirement.rb` file and, like the `JavaRequirement`, support using a `version` argument. This should perhaps be compulsory like the `RubyRequirement.